astro/src/content/articles/identity-basics/what-is-oidc.mdx

```diff
@@ -0,0 +1,170 @@
+
+<Aside type="note">
+A claim is a fancy term for a key/value pair in the JSON object which is the payload the JWT [Learn more about JWTs](/articles/tokens/).
+</Aside>
+
+There are a number of standard claims defined by the OIDC standard. Some reflect the authentication event and details about the OP. These include but are not limited to:
+
+* `iss`, the identifier of the issuer of the Id token. It's a URL like `https://example.com`.
+* `aud`, the identifier of the RP. Per the specification, it must be at least the `client_id` of the RP, but can include other identifiers.
+* `exp`, the expiration time of the Id token.
+* `iat`, the time at which the Id token was issued.
+* `auth_time`, the time the authentication event occurred.
+
+There are also user profile claims. Here's an incomplete list:
+
+* `sub`, an identifier for the end user at the OP.
+* `name`, the full name, in a format displayable to humans.
+* `preferred_username`, a preferred username for the RP to use. This is not guaranteed to be unique.
+* `email`, the user's preferred email address. Again, this is not guaranteed to be unique.
+* `email_verified` which is `true` if the email address has been verified to be controlled by the end user, as defined by the OP.
+* `locale`, the end user's locale, like `en-US` or `fr-CA`.
+* `phone_number`, the user's preferred telephone number.
+* `updated_at`, the time the user's information was last updated.
+
+All times are numbers representing the number of seconds since the unix epoch. The `iss`, `sub`, `aud`, `exp`, and `iat` claims are required and will be present in every Id token. Every other claim is optional, though some are required if the RP requests them or the OP is configured in certain ways. 
+
+You can see that an RP can get a decent amount of information about a user and the authentication from the OP. It's all transmitted in the Id token, which must be validated to ensure that there's no funny business going on.
+
+### Id Token Validation
+
+As an RP, you must [validate the Id token](https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation) you receive to ensure that the authentication event occurred at the correct OP in the correct manner. This is a multi step process:
+
+* verify the signature (or encryption if the Id token is encrypted).
+* ensure the `iss` claim matches the expected OP Issuer value.
+* ensure the `aud` claim contains the RP `client_id`.
+* validate that the time represented by the `exp` claim is in the future.
+
+Because of the flexibility of OIDC, there will be other validation steps depending on what data is requested.
+
+<Aside type="caution">
+The combination of the `iss` and `sub` claims is guaranteed by OIDC to be unique and stable. If you are building an application and need a unique identifier for a user, make sure you use the combination of these.
+
+The `sub` claim is guaranteed to be unique within any OP and never reassigned or changed.
+</Aside>
+
+### What Next?
```

Comment:
I don't love this h3 but I don't have a better one.

Reply:
Took another shot at it.